addons/scoreboard/mergedplayer.lua

```diff
@@ -234,8 +234,21 @@ function MergedPlayer:wsavg()
     end
 end
 
-
-
+function MergedPlayer:wsacc()
+    local hits, misses = 0, 0
+    
+    for _, p in ipairs(self.players) do
+        for _ in pairs(p.ws) do hits = hits + 1 end
```

Comment:
Ah thanks, that is much cleaner. I had googled this before in search of a general purpose `.length` accessor on the object itself and it seemed like all the answers pointed to doing a loop.